### ISSUE TITLE
Conan Build Failing

### ISSUE THREAD

**Issue by LeeRuns** [nanodbc/nanodbc#291]
## Environment

* nanodbc version: 2.13.0
* DBMS name/versin: sqlite None
* ODBC connection string: None 
* OS and Compiler: Windows Msys2 Mingw64 
* CMake settings: Conan build 2.13.0 with CPPSTD=17

## Actual behavior
I am trying to build this and it fails, because it cannot find numeric_limit.

## Expected  behavior
conan install should work 
## Minimal Working Example
here is the conan call from cmake 
```cmake
    conan_cmake_configure(REQUIRES
      boost/1.78.0
      sqlite3/3.38.0
      #gtest/cci.20210126
      nanodbc/2.13.0
      #nanodbc/cci.20200807 #FAIL - MINGW64 due to std::numeri_limits
      #poco/1.11.1          FAIL - MINGW64
      #soci/4.0.2
      #cpprestsdk/2.10.18
      #dataframe/1.19.0 FAIL GCC
      GENERATORS cmake_find_package
      #20220305  BUILD_REQUIRES cmake/3.22.0
      #VERSION 0 Less Greedy
      IMPORTS "bin, * -> ./bin"
      IMPORTS "lib, * -> ./bin"
      IMPORTS "include, *.h -> ./include")
```
### RESULTS IN THE FOLLOWING COMPILER ERROR
```cmake
nanodbc/2.13.0: Copying sources to build folder
nanodbc/2.13.0: Building your package in C:\Users\klr52\.conan\data\nanodbc\2.13.0\_\_\build\1338068473f48ac1f8814789ba77ba4d51096bdf
nanodbc/2.13.0: Generator cmake created conanbuildinfo.cmake
nanodbc/2.13.0: Aggregating env generators
nanodbc/2.13.0: Calling build()
-- The C compiler identification is GNU 11.2.0
-- The CXX compiler identification is GNU 11.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/msys64/mingw64/bin/gcc.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/msys64/mingw64/bin/g++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake targets configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan setting CPP STANDARD: 17 WITH EXTENSIONS OFF
-- Conan: Compiler GCC>=5, checking major version 11
-- Conan: Checking correct version: 11
-- Conan: C++ stdlib: libstdc++11
-- nanodbc version: 2.13.0
-- nanodbc compile: C++14
-- nanodbc build: Disable linking libc++ - True
-- nanodbc feature: ODBC Version Override - OFF
-- nanodbc feature: Disable async features - False
-- nanodbc feature: Enable Unicode - False
-- nanodbc feature: Enable Boost - False
-- nanodbc feature: Enable SQL_NO_DATA bug workaround - OFF
-- nanodbc build: Enable nanodbc target - STATIC
-- nanodbc build: Disable install target - False
-- nanodbc build: Disable tests target - True
-- nanodbc build: Disable examples target - True
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_INSTALL_BINDIR
    CMAKE_INSTALL_DATAROOTDIR
    CMAKE_INSTALL_INCLUDEDIR
    CMAKE_INSTALL_LIBDIR
    CMAKE_INSTALL_LIBEXECDIR
    CMAKE_INSTALL_OLDINCLUDEDIR
    CMAKE_INSTALL_SBINDIR


-- Build files have been written to: C:/Users/klr52/.conan/data/nanodbc/2.13.0/_/_/build/1338068473f48ac1f8814789ba77ba4d51096bdf
[ 50%] Building CXX object source_subfolder/CMakeFiles/nanodbc.dir/nanodbc/nanodbc.cpp.obj
C:\Users\klr52\.conan\data\nanodbc\2.13.0\_\_\build\1338068473f48ac1f8814789ba77ba4d51096bdf\source_subfolder\nanodbc\nanodbc.cpp: In function 'auto nanodbc::detail::from_string(const string&, R)':
C:\Users\klr52\.conan\data\nanodbc\2.13.0\_\_\build\1338068473f48ac1f8814789ba77ba4d51096bdf\source_subfolder\nanodbc\nanodbc.cpp:3392:24: error: 'numeric_limits' is not a member of 'std'
 3392 |     if (integer > std::numeric_limits<R>::max() || integer < std::numeric_limits<R>::min())
      |                        ^~~~~~~~~~~~~~
C:\Users\klr52\.conan\data\nanodbc\2.13.0\_\_\build\1338068473f48ac1f8814789ba77ba4d51096bdf\source_subfolder\nanodbc\nanodbc.cpp:3392:40: error: expected primary-expression before '>' token
 3392 |     if (integer > std::numeric_limits<R>::max() || integer < std::numeric_limits<R>::min())
      |                                        ^
C:\Users\klr52\.conan\data\nanodbc\2.13.0\_\_\build\1338068473f48ac1f8814789ba77ba4d51096bdf\source_subfolder\nanodbc\nanodbc.cpp:3392:46: error: no matching function for call to 'max()'
 3392 |     if (integer > std::numeric_limits<R>::max() || integer < std::numeric_limits<R>::min())
 
```

**Comment by mloskot** [comment]
Fixed by https://github.com/nanodbc/nanodbc/pull/279 in the current `master` branch.

**Comment by LeeRuns** [comment]
will this automatically fix the conan repo?

**Comment by mloskot** [comment]
No, unless Conan offers package based on the latest `master` branch. There are no automatic releases issued.
There is no nanodbc 2.14 yet (as you have seen, that PR is labelled for 2.14+).

@lexicalunit Is there anything that stops us from releasing 2.14?

**Comment by lexicalunit** [comment]
Let's do it!

**Comment by mloskot** [comment]
Let's celebrate then https://github.com/nanodbc/nanodbc/discussions/293

**Comment by LeeRuns** [comment]
Is the conan build working...

![image](https://user-images.githubusercontent.com/15914558/160244425-5294de33-9aae-45e5-8e02-c072cdbafc8f.png)

**Comment by mloskot** [comment]
I have no idea. You'll have to ask Conan folks. Speaking of nanodbc, the missing include has been fixed by #279
